tx-source/site_kb.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+$Definition[" by "] = " by ";
```

Comment:
I've deleted the file that contains this translation. We should remove it. https://github.com/vanilla/knowledge/pull/967